benchmarks/nyc_taxi/modin-ray/scale_cluster.py

```diff
@@ -0,0 +1,21 @@
+"""Request more cpus from the autoscaler and wait until they are available
+for compute tasks.
+
+usage:
+    ray submit modin-cluster.yaml scale_cluster.py
+"""
+
+import time
+
+import ray
+from ray.autoscaler.sdk import request_resources
+
+ray.init(address="auto")
+cpu_count = ray.cluster_resources()["CPU"]
+print("RAY CPU COUNT: ", cpu_count)
+
+request_resources(num_cpus=256)
```

Comment:
I assume 256 is the total vCPUs on the 4xc6i.16xlarge cluster and should not be physical cores?

Reply:
Yes